changelog/1825.feature.rst

```diff
@@ -0,0 +1 @@
+Moddified ~plasmapy.particles.particle_class artibutes to return numpy NaN when undefined.
```

Comment:
```suggestion
Modified ~plasmapy.particles.particle_class attributes to return NaN when undefined rather than raising exceptions.
```
I'll just apply this one and merge it.